require.js

```diff
@@ -60,7 +58,7 @@
   var initModule = function(name, definition) {
     var module = {id: name, exports: {}};
     cache[name] = module;
-    definition(module.exports, localRequire(name), module);
+    definition(module.exports, localRequire(name), module, globals);
```

Comment:
`global` is how to access the global scope in node.js. Some modules might use it and I thought this a better solution than assigning `window.global = window`. However if you'd like it removed I can do that.